src/status_im/contexts/wallet/send/select_address/tabs/view.cljs

```diff
@@ -46,16 +46,56 @@
                                                :stack-id  :screen/wallet.select-address}])}]))
             recent-recipients))))
 
+(defn- saved-address
+  [{:keys [name address chain-short-names customization-color ens? ens]}]
+  (let [full-address           (str chain-short-names address)
+        on-press-saved-address (rn/use-callback
+                                #(rf/dispatch
+                                  [:wallet/select-send-address
+                                   {:address   full-address
+                                    :recipient full-address
+                                    :stack-id  :screen/wallet.select-address}])
+                                [full-address])]
+    [quo/saved-address
+     {:user-props      {:name                name
+                        :address             full-address
+                        :ens                 (when ens? ens)
+                        :customization-color customization-color}
+      :container-style {:margin-horizontal 8}
+      :on-press        on-press-saved-address}]))
+
+(defn- saved-addresses
+  [theme]
+  (let [group-saved-addresses (rf/sub [:wallet/grouped-saved-addresses])
+        section-header        (rn/use-callback
```

Comment:
I think for `section-header` and  `empty-state-component` we should use `rn/use-memo`, they are view component and not function right?

Reply:
For `section-header`, it needs to function as it needs to receive title and index. So, use-callback is best here
For `empty-state-component`, it can be a memo. I can update it 👍